todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsContract.java

```diff
@@ -28,7 +28,7 @@
 
         void setProgressIndicator(boolean active);
 
-        void showStatistics(int numberOfIncompleteTasks, int numberOfCompletedTasks);
+        void showStatistics(long numberOfIncompleteTasks, long numberOfCompletedTasks);
```

Comment:
that's optimistic!

Reply:
Hmmm revert, I think it creates an unnecessary diff (makes merging harder)

Reply:
`.count()` was changed in RxJava 2 to return Long instead of count. I just update everywhere but I can change the contract so that the view takes `int`s